mcs/class/System.Net.Http/MonoWebRequestHandler.cs

```diff
@@ -0,0 +1,593 @@
+//
+// MonoWebRequestHandler.cs
+//
+// Authors:
+//	Marek Safar  <marek.safar@gmail.com>
+//      Martin Baulig  <mabaul@microsoft.com>
+//
+// Copyright (C) 2011-2018 Xamarin Inc (http://www.xamarin.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System.Collections.Generic;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Principal;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Specialized;
+using System.Net.Http.Headers;
+using System.Net.Cache;
+using System.Net.Security;
+using System.Linq;
+
+namespace System.Net.Http
+{
+	class MonoWebRequestHandler : IMonoHttpClientHandler
```

Comment:
This is the original HttpClientHandler, right? Can we please keep only a single copy of the file?

Reply:
It is - and I only see one copy of the file, where is the other one?